Sources/FoundationInternationalization/Formatting/Number/ICUNumberFormatter.swift

```diff
@@ -222,29 +222,28 @@ internal class ICUNumberFormatterBase : @unchecked Sendable {
 
 final class ICUNumberFormatter : ICUNumberFormatterBase, @unchecked Sendable {
     fileprivate struct Signature : Hashable {
-        let collection: NumberFormatStyleConfiguration.Collection
+        let skeleton: String
```

Comment:
Much cheaper to store the string here.

Reply:
There was previously a large cost of hashing all the values in the `Collection` when looking up formatters.